lib/auth_mvp_web/controllers/github_auth_controller.ex

```diff
@@ -1,13 +1,13 @@
 defmodule AuthMvpWeb.GithubAuthController do
   use AuthMvpWeb, :controller
-
   @elixir_auth_github Application.get_env(:auth_mvp, :elixir_auth_github) || ElixirAuthGithub
   @doc """
   `index/2` handles the callback from GitHub Auth API redirect.
   """
   def index(conn, %{"code" => code, "state" => client}) do
-    {:ok, _profile} = @elixir_auth_github.github_auth(code)
+    {:ok, profile} = @elixir_auth_github.github_auth(code)
+    jwt = AuthMvp.Token.generate_and_sign!(%{email: profile.email})
```

Comment:
Was `AuthMvp.Token.generate_and_sign` previous defined? (_it does not appear in this PR_)

Reply:
The jwt functions are imported in `AuthMvp.Token` module via the `use` of the `Joken.Config` module:
https://github.com/dwyl/auth-mvp/pull/4/files#diff-aeabf9b66d4c4f2b10a9da4bd106fb78

It is the recommended way to import these functions from Joken: https://hexdocs.pm/joken/configuration.html#module-approach
![image](https://user-images.githubusercontent.com/6057298/74823173-97358e00-5306-11ea-9105-40ae57536b68.png)

Reply:
Macro. 👍